zarr/core.py

```diff
@@ -2008,8 +1969,8 @@ def _chunk_getitems(self, lchunk_coords, lchunk_selection, out, lout_selection,
                 if ckey in self.chunk_store
             }
         else:
-            partial_read_decode = False
-            cdatas = self.chunk_store.getitems(ckeys, on_error="omit")
+            cdatas = self.chunk_store.getitems(ckeys, meta_array=self._meta_array)
```

Comment:
Since this is taking away code for the (previous) case that only some stores have `getitems`, can you please comment here with a link to the PR where this changed? Is it possible that there are old Store duck-types out there that lack this method, or do we now strictly require them to be subclasses of Store?

Reply:
AFAICT, all Stores should go through [`_ensure_store()`](https://github.com/zarr-developers/zarr-python/blob/main/zarr/_storage/store.py#L93) at some point.

Reply:
Also worth noting the `partial_read_decode = False` was just moved above the `if`. So it is still there (just always done now)